component/local/file/file.go

```diff
@@ -200,6 +200,16 @@ func (c *Component) Update(args component.Arguments) error {
 	if err := c.readFile(); err != nil {
 		return fmt.Errorf("failed to read file: %w", err)
 	}
+	return nil
+}
+
+// Update implements component.Component.
+func (c *Component) Update(args component.Arguments) error {
+
+	err := c.UpdateArgs(args)
+	if err != nil {
+		return err
+	}
```

Comment:
I think `Run()` could be called without calling `Update()`. In that case, is the fact that we didn't call `c.configureDetector()` prior to `Run()` a problem?